### PR TITLE
Daria/jg partnership

### PR DESCRIPTION
**Pull request by dariagrudzien**
Added a post announcing junior.guru partnership. The post is scheduled to go live on March 28th.

Preview: https://deploy-preview-37--infallible-sinoussi-4e1719.netlify.app/blog/partnership-with-junior-guru/

### CODE REVIEW

File: content/blog/partnership-with-junior-guru.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
++++
+title = "Partnership With junior.guru"
+description = "This next episode of our interview series includes a chat with Daria Grudzien - one of CyberMagnolia's Co-Founders."
```
Comment (anezkamll) [inline content/blog/partnership-with-junior-guru.md:3]: This is a leftover from the previous article I guess :) 
We can use something from the article, for example "We proudly announce our first partnership with an inspiring tech community junior.guru created by Honza Javorek."

File: content/blog/partnership-with-junior-guru.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+author = "Daria Grudzien"
+date = "2021-03-18"
+tags = ["partnership"]
+[[images]]
+  src = "/img/jg_partnership.jpeg"
+  alt = "Photo"
+  stretch = "stretchH"
++++
+
+Today we proudly announce of first partnership with an inspiring tech community **junior.guru** created by Honza Javorek.
```
Comment (anezkamll) [inline content/blog/partnership-with-junior-guru.md:13]: "we proudly announce of first partnership" sounds a bit weird to me, I believe the formulation is correct when announce is a noun but in this case as we use it as a verb so I would use maybe announce our first partnership... not sure, just feeling.

**Review by anezkamll — COMMENTED** [review]
Just two comments from my side.

File: content/blog/partnership-with-junior-guru.md (hunk 1 above)
Comment (dariagrudzien) [reply content/blog/partnership-with-junior-guru.md:3]: Good catch, thank you! <3

File: content/blog/partnership-with-junior-guru.md (hunk 2 above)
Comment (dariagrudzien) [reply content/blog/partnership-with-junior-guru.md:13]: I see, let me fix this up.

**Pull request merged**